mix.exs

```diff
@@ -23,7 +23,11 @@ defmodule AbsintheRelay.Mixfile do
       files: ["lib", "mix.exs", "README*"],
       maintainers: ["Bruce Williams", "Ben Wilson"],
       licenses: ["MIT"],
-      links: %{github: "https://github.com/absinthe-graphql/absinthe_relay"}
+      links: %{
+        "Website": "https://absinthe-graphql.org",
+        "Changelog": "https://github.com/absinthe-graphql/absinthe_relay/blob/master/CHANGELOG.md", 
```

Comment:
We should probably try to link to the individual version or tag changelog, since master is sometimes the changelog for the NEXT version.

Reply:
It seems like hex.pm shows the links from the last published version for all the versions, see for example https://hex.pm/packages/magixir/0.1.0 which did not have a changelog (nor a link to it) vs https://hex.pm/packages/magixir/0.1.1 (or https://hex.pm/packages/magixir) which added the link: both have exactly the same list of links.

For the main absinthe repo, I submitted 2 PR, the first one on master with the link pointing to the changelog on master branch (future v1.5) and another one on v1.4 branch with the corresponding changelog. In that case, because the changelog on the master branch does not contain the versions that are published (hum ... I should maybe also add a link from the 1.5 changelog on master branch to the 1.4 changelog to my PR, anyway, different story)

I got hooked by this idea of link from hex.pm to changelog file by https://hex.pm/packages/quantum which is the first package I saw doing that (with link to master) and since them I miss it on all the other packages I'm using.

The recommendation from https://keepachangelog.com/en/1.0.0/ seems to be to have a special `Unreleased` pseudo version on top of the file as placeholder for the next version.

And as a user, it doesn't really bother me if I have to scroll a bit to get to the version I'm interested in (if coming from an older version on hex.pm) or if I have to skip the `Unreleased` version. Actually `Unreleased` might be nice because we get a preview of what's coming...